### PR TITLE
Reduce cassette size

### PR DESCRIPTION
**Pull request by bgeuken**
The default file limit for diffs is 10.000. Since we only want to test
whether the diff truncation works or not, we can stub the diff size
limit with a much lower value (5).
This way we can reduce our test data and cassettes quite a bit.

### CODE REVIEW

File: src/api/app/models/bs_request_action/differ.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'bs_request_action/differ/for_source'
```
Comment (ChrisBr) [inline src/api/app/models/bs_request_action/differ.rb]: Why?

Comment (bgeuken) [reply src/api/app/models/bs_request_action/differ.rb]: Try running ./spec/controllers/webui/request_controller_spec.rb:128. You should get an exception (see commit message).

Comment (ChrisBr) [reply src/api/app/models/bs_request_action/differ.rb]: But I'm wondering why it worked before?

Comment (bgeuken) [reply src/api/app/models/bs_request_action/differ.rb]: I got this error when disabling VCR. Like I wrote;-)

Comment (ChrisBr) [reply src/api/app/models/bs_request_action/differ.rb]: Fine for me

**Pull request merged**